packages/portal-proto/src/features/facets/FacetSortPanel.tsx

```diff
@@ -74,22 +96,32 @@ const FacetSortPanel: React.FC<FacetSortPanelProps> = ({
         size="xs"
         compact
         color="primary.9"
-        onClick={() =>
-          setSort({
+        onClick={() => {
+          const direction =
+            sortType.type === "value" && sortType.direction === "asc"
+              ? "dsc"
+              : "asc";
+          const sortObj: SortType = {
             type: "value",
-            direction: sortType.direction === "asc" ? "dsc" : "asc",
-          })
-        }
+            direction,
+          };
+          setSort(sortObj);
+          setSortingStatus(
+            sortTypeToAriaDescription(sortObj, valueLabel, field),
+          );
+        }}
         rightIcon={<ValueSortIcon size={valueIconSize} />}
         aria-label={`Sort ${valueLabel} numerically`}
       >
         {valueLabel}
       </Button>
-      <span className="Offscreen">
-        <span id="liveRegion" aria-live="polite">
-          {sortTypeToAriaDescription(sortType, valueLabel)}
-        </span>
-      </span>
+
+      <span
+        id={`${field}-liveRegion`}
```

Comment:
is this Id needed? 

also this should not cause a problem but next time look into using react useId if you need to use an ID